client/packages/common/src/intl/locales/en/common.json

```diff
@@ -937,7 +939,9 @@
   "label.patient-id": "Patient ID",
   "label.patient-nuic": "NUIC",
   "label.patients": "Patients",
+  "label.patient-type": "Patient type",
   "label.percentage": "Percentage",
+  "label.pension-number": "Pension number",
```

Comment:
```suggestion
```
I'm not sure where this one came from, it's not actually in any of the requirements

Reply:
But is in some mockups, I don't think we'll actual end up using this!